test/arrays/ferguson/choose-array/choose-array3.chpl

```diff
@@ -5,10 +5,10 @@ proc chooseArray(A:[] int, B:[] int, chooseA:bool)
    else return B;
 }
 
-proc updateAndPrint(A:[], B:[], C:[], D:[])
+proc updateAndPrint(ref A:[], ref B:[], C:[], D:[])
 {
-  C[1] = 8;
-  D[1] = 9;
+  A[1] = 8;
+  B[1] = 9;
```

Comment:
I wonder if a better conversion would be to keep A, B const and instead give C, D the [non-const] in-intent.